transcom/mymove
# B 19157 int

[B-19157](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A923507)

## Summary

Is there anything you would like reviewers to give additional scrutiny?


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the customer application
2. Login as any role
3. Select "Privacy and Security Policy" at the bottom of the page
4. Verify the information is updated (screenshot)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2024-03-18 at 12 26 30 PM](https://github.com/transcom/mymove/assets/162157577/0478677f-e850-475c-b34f-6ec85f8efcee)
